packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -358,6 +427,33 @@ describe('Select', () => {
         await disconnect();
     });
 
+    it('when handling change event, value of select element matches what was selected in dropdown on focusout', async () => {
+        const { element, connect, disconnect } = await setup();
+        await connect();
+        await waitForUpdatesAsync();
+        const pageObject = new SelectPageObject(element);
+        await clickAndWaitForOpen(element);
+        let selectValue = '';
+
+        await Promise.race([
+            new Promise(resolve => {
+                element.addEventListener('change', () => {
+                    selectValue = element.value;
+                    resolve(true);
+                });
+                pageObject.pressArrowDownKey();
+                void (async () => {
+                    await pageObject.clickAway();
+                })();
+            }),
+            waitForUpdatesAsync().then(() => false)
+        ]);
```

Comment:
This seems a bit convoluted, why is the Promise race needed?

Can we have a helper to simplify this? Even if the following simplification isn't possible I want to avoid mixing promise and async patterns in the same block.

```ts
const changePromise = pageObject.waitForChange()
await pageObject.clickAway();
await changePromise;
selectValue = element.value;
```

Can also use [`once`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#options) to clean-up easier:

```ts
waitForChange () {
   return new Promise(resolve => {
      this.element.addEventListener('change', () => resolve(), {
         once: true
      });
   });
}
```